Content.Shared/Projectiles/EmbeddableProjectileComponent.cs

```diff
@@ -33,4 +34,10 @@ public sealed partial class EmbeddableProjectileComponent : Component
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite), DataField("offset"), AutoNetworkedField]
     public Vector2 Offset = Vector2.Zero;
+
+    /// <summary>
+    /// Sound to play after embedding into a hit target.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("sound"), AutoNetworkedField]
+    public SoundSpecifier? Sound;
```

Comment:
Can this just use the DamageOtherOnHit sound or is it different?

Reply:
afaik embed uses different code from that, the sound is played whenever it embeds which may or may not be the same as whenever it hits idk